careers/software-engineer.md

```diff
@@ -4,52 +4,56 @@ The goal of the Software Engineer path is to train engineers to master the funda
+**Productivity:** They reliably deliver the work assigned to them in a sprint. When they encounter a complex problem, they leverage the knowledge of their peers to find a suitable solution, even if it means changing the scope of the work. By the PR stage, their code is well-constructed; they do not expect others to catch sloppy mistakes for them.
 
-**Communication:** They communicate well and are capable of delivering feedback to peers and their manager. When given a task with unclear requirements they know how to ask for clarification, and ensure that all assumptions are vetted before work starts to reduce the need for re-work. They understand how their work fits into the larger picture for their team, and use this to identify conflicting requirements to their people leader and product manager. As Engineers learn they will start to participate more in the technical design process, often with guidance from senior engineers. Consistently speaks up in Sprint Planning meetings to ensure complete understanding of each work item. An Engineer II actively seeks out knowledge to supplement the team's weak points and areas of interest, including attending meetups, reading books, etc.
+**Impact:** They can take the lead on delivering work critical to meeting the sprint goal and are capable of independently planning work that multiple engineers can complete in a sprint or two. Instead of just executing on a task as written, they think critically about the problem and ensure that the solution will thoroughly solve the problem.
 
-Engineer IIs can take a bug or a task, and instead of just executing on the task as written, think critically about the problem and ensure that the issue has been thoroughly resolved by identifying root causes, proactively addressing performance concerns, providing proactive feedback on any product requirements and writing solid tests.
+**Communication:** They are able to effectively review code in their area of expertise and regularly point out opportunities for improvement in the code that they review. They proactively point out concerns with proposed or current sprint work as soon as they see them. They can articulate the tradeoffs of different approaches to solving a problem.
 
-By the time an engineer is ready to be promoted to Engineer III they will have focused on a technology or functional area as their expertise and become capable of mentoring interns and new engineers in these areas.
+**Collaboration:** They are mindful of the work that others are doing and can critique or suggest improvements to the proposed plans of others. They get help from others when stuck, but are also capable of helping unstick their peers. They are capable of effectively running small to medium-sized projects with regular coaching.
```

Comment:
Can we swap "they are mindful of" with something that describes visible behavior?

Reply:
"can critique or suggest improvements" reflects ability, rather than behavior. Do Engineers at this level engage in that behavior? What do our expectations look like?

Reply:
The behavior I'm trying to describe here is:

Going beyond just the work assigned to them, they are aware of what the rest of the team is working on. When they see an opportunity for improvement in someone else's work, they speak up.

Say they're working on a piece of backend work and hit a piece of complexity. They are aware of how the FE is planning to consume this data. They realize with a slight modification to their work, they could significantly simplify the FE implementation for another team member.